quisp/modules/QNIC/StationaryQubit/StationaryQubit.cc

```diff
@@ -1098,8 +1103,9 @@ measurement_outcome StationaryQubit::measure_density_independent() {
     }
   }
 
+  // if the partner qubit is measured,
```

Comment:
thanks for the useful comments. it would be better to make a method for checking it (e.g. `StationaryQubit::is_entanglement_partner_valid`) in other PR.

Reply:
Ah, that would be a good suggestion.  Let me create an issue about it.